src/Commands/RestoreCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the hyn/multi-tenant package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://laravel-tenancy.com
+ * @see https://github.com/hyn/multi-tenant
+ */
+
+namespace Hyn\Tenancy\Commands;
+
+use Hyn\Tenancy\Models\Website;
+use Illuminate\Console\Command;
+use Hyn\Tenancy\Traits\DispatchesEvents;
+use Hyn\Tenancy\Events\Websites as Events;
+
+class RestoreCommand extends Command
+{
+    use DispatchesEvents;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'tenancy:restore';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Restore all tenant databases present in the system db.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $websites = Website::all();
+
+        foreach ($websites as $website) {
+            $this->emitEvent(new Events\Created($website));
```

Comment:
I think we need to do a few things here;
- Check if the database already exists
- If not; create. If it does exists, we can get an exception here. 

Another option would be a try-catch and in the catch a message in the output that it failed to create the database.  Otherwise, the command would fail here and other websites won't be created as a new database.